config/master/aws-k8s-cni.yaml

```diff
@@ -70,6 +70,10 @@ rules:
     resources:
       - '*'
     verbs: ["list", "watch"]
+  - apiGroups: ["", "events.k8s.io"]
```

Comment:
This has to be updated in charts - https://github.com/aws/amazon-vpc-cni-k8s/blob/master/charts/aws-vpc-cni/templates/clusterrole.yaml as well and then generate the yaml.